tools/release.rb

```diff
@@ -0,0 +1,101 @@
+# encoding:utf-8
+
+# Copyright (c) 2013-2015 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+
+require_relative "release_checks"
+
+class Release
+  include ReleaseChecks
+
+  def initialize(opts = {})
+    @options = {
+      version:      generate_development_version
+    }.merge(opts)
+    @release_version = @options[:version]
+    @tag             = "v#{@release_version}"
+    @release_time    = Time.now.strftime('%a %b %d %H:%M:%S %Z %Y')
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.